doc/CMakeLists.txt

```diff
@@ -10,46 +60,37 @@ endif()
 
 find_package(Doxygen 1.8.0 ${REQUIRED})
 
-if(DOXYGEN_FOUND)
```

Comment:
Can get rid of the `if() ... "REQUIRED"` bit above (can't comment on unchanged lines…) and just `find_package` inside `if(BUILD_DOXYGEN)`.